istioctl/cmd/proxystatus.go

```diff
@@ -103,14 +111,26 @@ Retrieves last sent and last acknowledged xDS sync from Istiod to each Envoy in
 			if err != nil {
 				return err
 			}
+
 			sw := pilot.StatusWriter{Writer: c.OutOrStdout()}
+
+			switch statusArgs.output {
+			case jsonFormat:
+				return sw.PrintJSON(statuses)
+			case tableFormat:
```

Comment:
Is there no processing done here?

Reply:
I based this on what I saw in another part of the codebase https://github.com/istio/istio/blob/master/istioctl/cmd/tag.go#L413-L419. In the case `tableFormat` is used (or no specified as its the default) it will fall through to line 125 and `PrintAll` as is the old behavior.